File: Snake_Water_Gun.py
# Snake, Water, Gun game
import random

computerChose = random.choice(['w', 's', 'g'])
yourChoice = input("Enter Your choice ( w / s / g ) : ")

dict = {'w':"Water", 's':"Snake", 'g':"Gun"}

computer = dict[computerChose]
you = dict[yourChoice]

print(f" Computer Chose : {computer} \n You Chose : {you}")

if (computerChose == yourChoice) :
    print("Draw!")
else :
    if (computerChose == 'w' and yourChoice == 's'):
        print("You Win!")
    elif (computerChose == 'w' and yourChoice == 'g'):
        print("You Lose!")
    elif (computerChose == 'g' and yourChoice == 's'):
        print("You Lose!")
    elif (computerChose == 'g' and yourChoice == 'w'):
        print("You Win!")
    elif (computerChose == 's' and yourChoice == 'w'):
        print("You Lose!")
    elif (computerChose == 's' and yourChoice == 'g'):
        print("You Win!")
    else :
        print("Something went wrong!")

File: Snake_Water_Gun.java
package Mini_Projects;

import java.util.Scanner;
import java.util.Random;
import java.util.HashMap;

public class Snake_Water_Gun{
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        HashMap<Character, String> map = new HashMap<>();

        char[] options = {'w', 's', 'g'};
        map.put('w', "Water");
        map.put('g',"Gun");
        map.put('s',"Snake");
        // Random random = new Random();
        char computerChose = options[new Random().nextInt(options.length)];

        System.out.print("Enter Your Choice (s,g,w) : ");
        char youChose = sc.next().charAt(0);

        System.out.println(" Computer Chose : "+map.get(computerChose)+"\n You Chose : "+map.get(youChose));

        if (computerChose == youChose) {
            System.out.println("Drow!");
        } else {
            if (computerChose == 'g' && youChose == 'w') {
                System.out.println("You Win!");
            } else if (computerChose == 'g' && youChose == 's') {
                System.out.println("You Lose!");
            } else if (computerChose == 'w' && youChose == 's') {
                System.out.println("You Win!");
            } else if (computerChose == 'w' && youChose == 'g') {
                System.out.println("You Lose!");
            } else if (computerChose == 'g' && youChose == 's') {
                System.out.println("You Lose!");
            } else if (computerChose == 'g' && youChose == 'w') {
                System.out.println("You Win!");
            } else {
                System.out.println("Something went Wrong!");
            }
        }
    }
}